tests/functional/test_accounts.py

```diff
@@ -30,7 +30,7 @@ def test_submit_email_form_with_xhr_returns_partial_html_snippet(self,
         email_form['email_confirm'] = 'new_email@example.com'
         email_form['password'] = 'pass'
 
-        res = email_form.submit(xhr=True)
+        res = email_form.submit(xhr=True, status=200)
```

Comment:
This line already would have crashed if the response status code was non-successful, here I'm just making it explicit that that is tested.